doc/src/vpicUnits.tex

```diff
@@ -37,41 +51,156 @@
+	\end{equation*}
+
+	The deck constants are therefore
+
+	\begin{align*}
+		l_{\mathrm{deck}} &= \ldebye, &
+		c_{\mathrm{deck}} &= \frac{c}{\vth}, \\
+		\varepsilon_{0,\mathrm{deck}} &= 1, &
+		r_{e,\mathrm{deck}} &= -1,
+	\end{align*}
+
+	\noindent giving the unit conversions as
+
+	\begin{align*}
+		L &= \ldebye, &
+		T &= \omegap^{-1}, &
+		M &= N_{\mathrm{D}}m_e, \\
+		Q &= N_{\mathrm{D}}e, &
+		E &= \frac{m_e}{e}\vth\omegap, &
+		B &= \frac{m_e}{e}\omegap,
+	\end{align*}
+
+	\noindent where $N_{\mathrm{D}} \equiv n_e\ldebye^3$ is the number of electrons in a `Debye cube'.
+
+	This unit system allows for some convenient sanity checks. As mentioned above, the cell size and time-step must be around $1$. Additionally, if the electron mass in these units is not significantly smaller than 1, then the plasma is not ideal and simulating it with VPIC is not appropriate. Interpretation of simulation results may also be more convenient. For example the normalised electric field $E_c$ is the electron quiver velocity in a plasma wave divided by the thermal velocity - an important parameter for determining whether wavebreaking will occur. Similarly the wavenumber of a plasma wave is normalised such that $k_c = k\ldebye$, which determines the Landau damping rate.
+
+	\subsubsection{Electromagnetic Plasma Units}
+
+	For an electromagnetic problem, the electrostatic units above might be modified so that the length-scale becomes the electron skin depth $c/\omegap$. The Lorentz force and Maxwell's equations become
+
+	\begin{equation*}
+		\ddot{\vec{r}} = \vec{E} + \vec{v} \times \vec{B},
+	\end{equation*}
+	\noindent and
+	\begin{align*}
+		\nabla \times \vec{E} &= -\partial_t \vec{B}, \\
+		\nabla \times \vec{B} &= \left(\vec{J} + \partial_t \vec{E}\right).
+	\end{align*}
+
+	\noindent The deck constants are then
+
+	\begin{align*}
+		l_{\mathrm{deck}} &= \frac{c}{\omegap}, &
+		c_{\mathrm{deck}} &= 1, \\
+		\varepsilon_{0,\mathrm{deck}} &= 1, &
+		r_{e,\mathrm{deck}} &= -1,
+	\end{align*}
+
+	\noindent so that the conversion factors are now
+
+	\begin{align*}
+		L &= \frac{c}{\omega_{\mathrm{pe}}}, &
 		T &= \frac{1}{\omega_{\mathrm{pe}}}, \\
-		M &= \varepsilon_0\frac{c^3}{\omega_{\mathrm{pe}}}\frac{m_e^2}{e^2}, \\
-		Q &= \varepsilon_0\frac{c^3}{\omega_{\mathrm{pe}}}\frac{m_e}{e}, \\
-		E &= \frac{m_e\omega_{\mathrm{pe}}c}{e}, \\
-		B &= \frac{m_e\omega_{\mathrm{pe}}}{e}
-	\end{align}
+		M &= n_e\frac{c^3}{\omegap^3}m_e, &
+		Q &= n_e\frac{c^3}{\omegap^3}e, \\
+		E &= \frac{m_e}{e}\omegap c, &
+		B &= \frac{m_e}{e}\omegap.
+	\end{align*}
 
-	\noindent Note here that the system of units will depend on the electron density used to calculate $\omega_{\mathrm{pe}}$. A more practical alternative for problems involving lasers is to choose $l_{\mathrm{deck}}=c/\omega_0$ where $\omega_0$ is the laser frequency, since it is more common to need to change density than the laser frequency.
+	\noindent Note that there is no longer any dependence on temperature.
```

Comment:
This is the first mention of temperature.  If it's absence is important then you should say where it is present (Debye length).

Reply:
Good point - I'll add that.